### DISCUSSION TITLE
[Question] What does "Activate tab on mouse button release" do exactly?

### DISCUSSION THREAD

**Discussion by fourteen-1** [mbnuqw/sidebery#977] [Q&A]
I can't seem to figure out what this feature actually is. Could someone help me out?

**Comment by mbnuqw** [comment]
It allows you to choose the event of the mouse click (button press or button release) after which tab will be activated. E.g. activating a tab on button-release is useful when you want to drag a tab with mouse without activation of that tab.

**Reply by fourteen-1** [reply]
By "activate" do you mean it will open/switch to that tab? because toggling this feature on/off doesn't seem to have any effect when dragging a tab.

**Reply by fourteen-1** [reply]
Edit: Never mind. It wasn't functioning correctly because I had "Long press to Duplicate tab" feature on. Thanks for the help!
